modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,106 @@
+[[clients-oracle]]
+= Registering {oracle} Clients
+
+This section contains information about registering traditional and Salt clients running {oracle} operating systems.
+
+ifeval::[{suma-content} == true]
+[IMPORTANT]
+====
+{oracle} repository URLs are available from {scc}, but the packages and metadata are provided by Oracle, not by {suse}.
+{oracle} base media repositories are freely available for download from https://yum.oracle.com/.
+====
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+
+== Channel and Repository Management
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
```

Comment:
I assume the data was provided by Kevin, for 6, 7 and 8, right?

If so, then this is fine :-)

Reply:
Yes, it was.